Rubberduck.Main/Root/RubberduckIoCInstaller.cs

```diff
@@ -147,31 +149,48 @@ public void Install(IWindsorContainer container, IConfigurationStore store)
 
             RegisterSpecialFactories(container);
             RegisterFactories(container, assembliesToRegister);
-            
+
             ApplyDefaultInterfaceConvention(container, assembliesToRegister);
         }
 
+        private void RegisterUnitTestingComSide(IWindsorContainer container)
+        {
+            container.Register(Component.For<IFakesFactory>().AsFactory());
```

Comment:
nitpick: As stated in the introductory comment to the RubberduckIoCInstaller, I would prefer if the singleton scope was made explicit.